journalpump/journalpump.py

```diff
@@ -457,6 +483,12 @@ def run(self):
             entry = None
             try:
                 self.initialize_sender()
+                msg_buffer_length = len(self.msg_buffer)
+                if msg_buffer_length > 50000:
+                    self.log.debug("%d entries in msg buffer, slowing down a bit by sleeping",
+                                   msg_buffer_length)
+                    time.sleep(1.0)
```

Comment:
Is there reason why we wouldn't stop reading new entries altogether if we're unable to flush our buffer?